src/components/Accordion/Accordion.reducers.spec.js

```diff
@@ -1,5 +1,5 @@
 import assert from 'assert';
-import * as reducers from './Accordion.reducers.js';
+import * as reducers from './Accordion.reducers.ts';
```

Comment:
I think you can leave off the file extension type here, so just `'./Accordion.reducers';